tap_kafka/sync.py

```diff
@@ -242,6 +252,8 @@ def assign_consumer_to_timestamp(consumer, topic: str, timestamp: str, timeout:
     # Assign to the correct position
     consumer.assign(partitions_to_assign)
 
+    consumer = seek_partitions(consumer, partitions_to_assign)
```

Comment:
1. new `consumer` variable is not used at all here therefore no need for assignment here.
2. As I wrote in the previous comment, the input `consumer` is manipulated already after calling `seek_partitions`